build.gradle

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id 'java'
+    id 'idea'
+    id 'org.springframework.boot' version '2.1.7.RELEASE'
+    id 'io.spring.dependency-management' version '1.0.8.RELEASE'
+}
+
+group 'org.neorg'
+version '1.0-SNAPSHOT'
```

Comment:
The very first version of a project is recommended to be set like "0.0.1".
Where:
- "0.X.X" - 0 is Major version. It's incremented when you release new functionality and fix public API.
- "X.0.X" - 0 is Minor version.
- "X.X.1" - 1 is Patch version.
P.S. There could be an additional version number like "Build version" and "Compilation version" - 0.0.1.5.133.
If you use intermediate version - should be used word "SNAPSHOT".

More info here: https://semver.org/